ui/src/frontend/time_selection_panel.ts

```diff
@@ -128,22 +130,46 @@ function drawIBar(
 
 export class TimeSelectionPanel extends Panel {
   view() {
-    return m('.time-selection-panel');
+    return m('.time-selection-panel', {
```

Comment:
This is the third occurrence (so it seems to me) of this new addition of mouse handling. Can this be shared somehow instead of repeating it, for maintainability? Especially considering that we'll eventually need to port this onto the latest Perfetto upstream (unless they already have implemented resizable track headers themselves).